### ISSUE TITLE
Delete gitignored files command

### ISSUE THREAD

**Issue by Marr11317** [Axosoft/vscode-gitlens#1705]
Sometimes something that looks and feels like a clean repo is wanted, and this is most often achieved by deleting all .gitignored files.

In node projects, this often results in deleting node_modules/ folder, and the output directetory.

For c/cpp projects, this often means deleting the cmake-generated files.

These files / folder have a thing in common: they are often in .gitignore, and it often happens that I'd like to delete all files that are .gitignored, to get something that looks like a clean install.

# TLDR

I would like a command `GitLens: delete ignored files`.

Note that this command would delete _all_ files. Including `.env` and any other preference files.

Any thoughts?
